conf/config.yaml

```diff
@@ -16,7 +16,7 @@
 #
 apisix:
   node_listen: 9080              # APISIX listening port
-  enable_heartbeat: true
+  enable_heartbeat: false
```

Comment:
we need to remove this line.

some code we need to remove too: 

https://github.com/apache/incubator-apisix/search?p=2&q=enable_heartbeat&unscoped_q=enable_heartbeat

Reply:
Thanks for taking time to review, will do.